pkg/diagnostics/diagnostic_bundle_test.go

```diff
@@ -462,6 +462,9 @@ func TestGenerateManagementClusterBundle(t *testing.T) {
 						Host: "1.1.1.1",
 					},
 				},
+				DatacenterRef: eksav1alpha1.Ref{
+					Kind: "VSphereDatacenterConfig",
```

Comment:
can we add a test for docker data center config and confirm that it does not get triggered for docker?

Reply:
I did try to test for docker but since we need test two spec in this unit test, I couldn't make it work. Is it possible to test this in a separated unit test?